app/src/main/res/values/attrs_tiles.xml

```diff
@@ -14,11 +13,9 @@
     <attr name="tile_artists_bg" format="color" />
     <attr name="tile_audio_bg" format="color" />
     <attr name="tile_chapter_bg" format="color" />
-    <attr name="tile_edit_bg" format="color" />
     <attr name="tile_genres_bg" format="color" />
     <attr name="tile_land_folder_bg" format="color" />
     <attr name="tile_land_photo_bg" format="color" />
-    <attr name="tile_land_series_timer_bg" format="color" />
```

Comment:
This one should be used in `tile_land_series_timer.xml` but it uses the wrong one there. Can you fix that in this PR? (probably rename the other one to `tile_land_series_timer_bg` now)

Reply:
Good catch. It looks like both port and land are needed.